lib/p2p/Pool.ts

```diff
@@ -93,6 +97,12 @@ class Pool extends EventEmitter {
         const address = addressUtils.fromString(addressString, config.port);
         this.addresses.push(address);
       });
+      config.blacklist.forEach((addressString) => {
+        this.blacklist.push(addressString);
+      });
+      config.whitelist.forEach((addressString) => {
+        this.whitelist.push(addressString);
```

Comment:
No need to loop through and `push` every element into a new array, you can just do `this.whitelist = config.whitelist;`.